language/stdlib/modules/lbr.move

```diff
@@ -94,6 +97,7 @@ module LBR {
         let coin2_exact = Libra::withdraw(&mut coin2, num_coin2);
         Libra::deposit(&mut reserve.coin1.backing, coin1_exact);
         Libra::deposit(&mut reserve.coin2.backing, coin2_exact);
+        // TODO: This will emit an event. Do we want that for LBR?
```

Comment:
I think this TODO was resolved in this PR, so can probably remove?

Reply:
Good catch. I wrote the TODO and then discovered the magic of `is_synthetic`. Whoever introduced that is a genius.